src/domain/consents/ID.ts

```diff
@@ -77,29 +77,31 @@ export async function updateConsentCredential (
   return consentDB.update(consent)
 }
 
-export async function buildConsentRequestBody (
-  consent: Consent,
-  signature: string,
-  publicKey: string): Promise<PutConsentsRequest> {
+/* TODO: complete this function. only updated it to conform to the
+         API interface for now and not for actual use!
+*/
+export async function buildConsentsIDPutResponseVerifiedBody (
+  consent: Consent
+  ): Promise<tpAPI.Schemas.ConsentsIDPutResponseVerified> {
   /* Retrieve the scopes pertinent to this consentId
    and populate the scopes accordingly. */
   const scopes: Scope[] = await scopeDB.retrieveAll(consent.id)
-  const externalScopes: ExternalScope[] = convertScopesToExternal(scopes)
+  const TPScopes: tpAPI.Schemas.Scope[] = convertDatabaseScopesToThirdpartyScopes(scopes)
 
-  const consentBody: PutConsentsRequest = {
-    requestId: consent.id,
-    scopes: externalScopes,
-    initiatorId: consent.initiatorId as string,
-    participantId: consent.participantId as string,
+  const consentBody: tpAPI.Schemas.ConsentsIDPutResponseVerified = {
+    scopes: TPScopes,
     credential: {
-      id: consent.credentialId as string,
       credentialType: 'FIDO',
       status: CredentialStatusEnum.VERIFIED,
-      challenge: {
-        payload: consent.credentialChallenge as string,
-        signature: signature
-      },
-      payload: publicKey
+      payload: {
+        id: consent.credentialId!,
+        rawId: 'TODO: figure out what goes here',
```

Comment:
`rawId` is a byte array made from string  `id`

Reply:
Ah. Thank you. I'll get that in my pass that I'm working on today.